content/en/docs/concepts/architecture/nodes.md

```diff
@@ -275,6 +275,12 @@ and do not respect the unschedulable attribute on a node. This assumes that daem
 the machine even if it is being drained of applications while it prepares for a reboot.
 {{< /note >}}
 
+{{< note >}}
```

Comment:
Does this actually merit a 
```suggestion
{{< caution >}}
```
?

Reply:
Possibly, but I'm 100% new to doc-writing (and didn't know the tag), so I'll pass that question on to the reviewers and update according to the answer.

Reply:
@MMeent no worries.
I recommend switching from `{{< note >}}` to `{{< caution >}}`.

Reply:
Sorry for the late reply, applied feedback